### ISSUE TITLE
[GCD] How to go from SPH to a field of physical properties?

### ISSUE THREAD

**Issue by MGJvanGroeningen** [Joshuaalbert/neural_deprojection#21]
- [ ] 1 What is SPH? How is it used in simulations?
- [ ] 1 Understand physical conversion from SPH to physical quantities
- [ ] Create/understand code that can convert SPH to physical quantities

**Comment by Joshuaalbert** [comment]
The internal energy U is :
```
function U2T(unit::CodeUnits, U::Array; xH=0.76, gam=5/3, rad=false)

   yhelium = (1-xH)/(4*xH)

   mean_mol_weight = (1+4*yhelium)/(1+3*yhelium+1)

   return U .* (gam-1) .* unit.vel^2 .* mp .* mean_mol_weight ./ k_boltz
```

**Comment by Joshuaalbert** [comment]
Merged with #20
